packages/react-server-cli/src/ConfigurationError.js

```diff
@@ -0,0 +1,13 @@
+// Can't use `instanceof` with babel-ified subclasses of builtins.
+//
+//   https://phabricator.babeljs.io/T3083
+//
+// Gotta do this the old-fashioned way. :p
+//
+export default function ConfigurationError(message) {
```

Comment:
We might want to consider using [VError](https://github.com/davepacheco/node-verror) for our custom error classes